internal/converter/internal/staticconvert/internal/build/snmp_exporter.go

```diff
@@ -64,15 +64,15 @@ func (b *ConfigBuilder) appendSnmpExporterV2(config *snmp_exporter_v2.Config) di
 }
 
 func toSnmpExporterV2(config *snmp_exporter_v2.Config) *snmp.Arguments {
-	targets := make([]snmp.SNMPTarget, len(config.SnmpTargets))
-	for i, t := range config.SnmpTargets {
-		targets[i] = snmp.SNMPTarget{
-			Name:       common.SanitizeIdentifierPanics(t.Name),
-			Target:     t.Target,
-			Module:     t.Module,
-			Auth:       t.Auth,
-			WalkParams: t.WalkParams,
-		}
+	var targets snmp.TargetsList
+	for _, t := range config.SnmpTargets {
+		target := make(map[string]string)
+		target["name"] = common.SanitizeIdentifierPanics(t.Name)
```

Comment:
we should be able to remove the call to `common.SanitizeIdentifierPanics` now that we are not mapping to a block label. let's include in our test something that is invalid river to double check this.

Reply:
good catch, I made the change and added a "." in the name in the yml file. The "." would be replace by a "_" if we would still use SanitizeIdentifierPanics